lib/rucio/core/transfer.py

```diff
@@ -1260,8 +1258,8 @@ def get_transfer_requests_and_source_replicas(total_workers=0, worker_number=0,
 
 
 @read_session
-def __list_transfer_requests_and_source_replicas(total_workers=0, worker_number=0,
-                                                 limit=None, activity=None, older_than=None, rses=None, session=None):
+def __list_transfer_requests_and_source_replicas(total_workers=0, worker_number=0, limit=None, activity=None,
+                                                 older_than=None, rses=None, request_state=None, session=None):
```

Comment:
This method is called by `get_transfer_requests_and_source_replicas`, but you changed the signature by introducing a request_state. The call should be changed in `get_transfer_requests_and_source_replicas`

Reply:
will do this anyway, although it works fine with the default

Reply:
just saw this `request_state=RequestState.QUEUED` in the `get_transfer_requests_and_source_replicas` call …? Nothing to do here.